File: strategy/ducks/Dance/Manuette.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text;

namespace ducks.Dance
{
	public class Manuette : IDanceBehaviuor
	{
		public Manuette(StringBuilder stringBuilder) => _stringBuilder = stringBuilder;

		public void Dance() => _stringBuilder?.Append("manuette");

		private readonly StringBuilder _stringBuilder;
	}
}

File: strategy_java/src/ducks/strategies/Manuette.java
package ducks.strategies;

import ducks.base.IDanceBehaviour;

public class Manuette implements IDanceBehaviour {
    public Manuette(StringBuilder sb) {
        _sb = sb;
    }

    @Override
    public void Dance() {
        _sb.append("manuette");
    }

    private final StringBuilder _sb;
}
